analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/ControllingPermissions.CSharp11.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Threading;
+using Microsoft.AspNetCore.Http;
+
+[Obsolete(nameof(httpContext.User))] // Compliant, because HttpContext class was moved from System.Web to Microsoft.AspNetCore.Http and we do not track it
+void HttpContextUser(HttpContext httpContext)
```

Comment:
The table is not populated for this rule. You need to specify also "Implement" to fix the FNs and the FP below.